torch/_inductor/fx_passes/quantization.py

```diff
@@ -134,6 +134,27 @@ def _may_generate_pattern_with_dtype_convert(pattern, dtype=Arg(), dtype_convert
     KeywordArg("postop_algorithm"),
 )
 
+
+def get_qlinear(users=1):
```

Comment:
Suggest to use `get_qlinear` for where using `qlinear_pt2e_pattern`, since they can share the code.

Reply:
Done.

Reply:
`qlinear_pt2e_pattern` is better?

Reply:
Thanks for the comment! I've changed to `qlinear_pt2e_pattern`.